src/components/flat-table/flat-table-expandable.stories.mdx

```diff
@@ -175,6 +238,88 @@ Only clicking on the first column can expand the row with this option set.
   </Story>
 </Preview>
 
+### Expandable by first column only keyboard accessible subrow
```

Comment:
I don't think this story provides any value that the other new one doesn't, it can be removed as to not muddy up the docs